index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
https://github.com/mrEckendonk/todo-today/blob/47607ac7d58f365e62cc2eb29658435ae06d8e50/package.json#L6-L13

- [ ] After installing your dependencies by running `npm install`, Then I try running `npm run start` but the app could not be open into the browser. I checked your `package.json` file and found out that the `start` script which is responsible for starting your app in the browser is missing, I added the `start` script `"start": "webpack serve --open"` and I was able to run your app with `npm run start`. Kindly I strongly recommend you to add the start script in the package.json.

Reply:
Strange, I have the `start` in on line 7, and with a Windows machine, it works fine.
![image](https://user-images.githubusercontent.com/30936842/159716256-1f0f8d41-50c0-4ca7-9b41-25b15562cb10.png)